scripts/utility.py

```diff
@@ -2970,10 +2970,15 @@ def quit(savesettings=False, clearevents=False):
     sys_exit()
 
 
+resource_directory = "resources/dicts/conditions/"
+with open(os.path.normpath(f"{resource_directory}illnesses.json"), "r", encoding="utf-8") as read_file:
+    ILLNESSES = ujson.loads(read_file.read())
+
+with open(os.path.normpath(f"{resource_directory}injuries.json"), "r", encoding="utf-8") as read_file:
+    INJURIES = ujson.loads(read_file.read())
+
```

Comment:
...where was this before and why is it loaded here rather than the places it's used?

Reply:
circular imports were killing me, for some reason after i merged in the localization stuff, it started yelling that accessing those files from cat.py caused a circular import (it didn't before? im sobbing).  PERMANENT was already being loaded here anyways so i dropped the other ones in too.  I know it's not particularly ideal ^^; would you rather i put them in herb_supply?

Reply:
Oh huh. Nah, leave them all here.